configstack/running_module.go

```diff
@@ -104,9 +105,27 @@ func crossLinkDependencies(modules map[string]*runningModule, dependencyOrder De
 		}
 	}
 
+	removeFlagExcluded(modules)
```

Comment:
This should probably not be called in `crossLinkDependencies` but separately to avoid confusion.

Reply:
moved this to `toRunningModules()` which should separate the actions more clearly.